### DISCUSSION TITLE
Configuring the WebCrawser as Data Provider

### DISCUSSION THREAD

**Discussion by superbobich** [dachcom-digital/pimcore-dynamic-search#68] [Q&A]
Hi, I would be very thankful if someone could help me. 
I am using PIM X and I'm trying to get Dynamic Search to run. 
For a Data Provider I would like to use the webCrawler. 
The thing is in the documentation there are things like the Trasnformers, more scpecifically the scaffolders and the field transformers and I have no clue where to configure them in the yaml. 
I tried to bring them under the data_provider but sadly it doesn't work. 
Would it be possible for someone to post an example of a full yaml configuration with everything in it? 
Would really appreciate if someone with more experience could help me out.

**Comment by beheist** [comment]
I have exactly the same problem. This package seems like it could work very well, but without explaining the concepts and how to actually implement them, I cannot figure out how to correctly configure it.
Have asked the same question over in the main repo.
https://github.com/pimcore/pimcore/discussions/15997

**Comment by solverat** [comment]
@superbobich, @beheist maybe this PR (which hasn't been merged by now) will guide you into the right direction: https://github.com/dachcom-digital/pimcore-dynamic-search/pull/66

**Comment by beheist** [comment]
Thank you Stefan, that looks like it might be helpful. We will see if that helps us along further.
